pom.xml

```diff
@@ -86,5 +86,11 @@
             <version>2.8.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Shading an annotations lib again, which is already provided, is redundant.

I suggest using the one already [provided](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/pom.xml#80).